index.html

```diff
@@ -0,0 +1,571 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
```

Comment:
`HINT`

Consider avoiding the use of  `maximum-scale=1.0` and `user-scalable=0` in the viewport meta tag. This can lead to accessibility issues such as preventing zoom settings from working on a browser. 

Learn more [here](https://a11yproject.com/posts/never-use-maximum-scale/)